spec/inertia/conditional_sharing_spec.rb

```diff
@@ -14,6 +14,20 @@
       expect(JSON.parse(response.body)['props'].deep_symbolize_keys).not_to include({
         conditionally_shared_show_prop: 1,
       })
+      get conditional_share_edit_path, headers: { 'X-Inertia' => true }
+      expect(JSON.parse(response.body)['props'].deep_symbolize_keys).to eq({
```

Comment:
I think this spec is solid, but could we add one more spec here that verifies that these props are **not** included when a different action on this controller is requested? I always like covering the opposite of the happy path as well since I believe this spec would also pass if I deleted all of the filtering code.

Reply:
Updated the spec to use `expect.to eq` and replaced the props names to make them more obvious.

Reply:
awesome thank you; new spec looks great!